### PR TITLE
[DRUP-720] [DRUP-756] Remove deprecated code from Apigee Kickstart

### PR DESCRIPTION
**Pull request by shadcn**
Now that https://github.com/apigee/apigee-edge-drupal/pull/179 is in, we can remove the fix for it in kickstart.

### CODE REVIEW

**Comment by jacine** [comment]
I think we need to wait until this is released in apigee_edge to merge this. Lmk if that's wrong.

**Comment by shadcn** [comment]
> I think we need to wait until this is released in apigee_edge to merge this. Lmk if that's wrong.

Yeah this can wait now that Kickstart is using stable releases: https://github.com/apigee/apigee-devportal-kickstart-drupal/commit/87e80b93b5024f7ec4b6be3b58fa05533310d627#diff-b5d0ee8c97c7abd7e3fa29b9a27d1780

**Review by jacine — APPROVED** [review]
Thanks @arshad!  This looks good.

**Pull request merged**
